homeassistant/components/yeelight/__init__.py

```diff
@@ -472,7 +469,7 @@ def turn_off(self, duration=DEFAULT_TRANSITION, light_type=None):
             self.bulb.turn_off(duration=duration, light_type=light_type)
         except BulbException as ex:
             _LOGGER.error(
-                "Unable to turn the bulb off: %s, %s: %s", self.ipaddr, self.name, ex
+                "Unable to turn the bulb off: %s, %s: %s", self.host, self.name, ex
```

Comment:
There are four places in this class where we use `self.host` and three places where we use `self._host`. I suggest to be consistent and use `self._host` inside the class. Use the property for external access.
```suggestion
                "Unable to turn the bulb off: %s, %s: %s", self._host, self.name, ex
```